man/org.freedesktop.portable1.xml

```diff
@@ -309,7 +309,8 @@ node /org/freedesktop/portable1 {
       <function>ReattachImageWithExtensions()</function> methods take in options as flags instead of
       booleans to allow for extendability. <varname>SD_SYSTEMD_PORTABLE_FORCE</varname> will cause
       safety checks that ensure the units are not running while the new image is attached or detached
-      to be skipped. They are defined as follows:</para>
+      to be skipped, and do not insist that the extension-release.NAME file in the extension image has to
```

Comment:
as mentioned earlier, could use some `<filename>` and `<replaceable>` tag love

Reply:
I'll add `filename`, but what should `replaceable` be used for?

Reply:
NAME